dependencies/shotgun_api3/lib/simplejson/decoder.py

```diff
@@ -269,7 +273,8 @@ def JSONObject((s, end), encoding, strict, scan_once, object_hook,
         pairs = object_hook(pairs)
     return pairs, end
 
-def JSONArray((s, end), scan_once, _w=WHITESPACE.match, _ws=WHITESPACE_STR):
+def JSONArray(xxx_todo_changeme1, scan_once, _w=WHITESPACE.match, _ws=WHITESPACE_STR):
```

Comment:
and here?

Reply:
Tuple parameters are no longer support in Python3: http://www.python.org/dev/peps/pep-3113/